src/pages/utkast/skjema/begrunnelse/begrunnelse.tsx

```diff
@@ -11,8 +11,11 @@ import { useDataStore } from '../../../../stores/data-store';
 import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
 import { OrNothing } from '../../../../utils/types/ornothing';
 import './begrunnelse.less';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import Show from '../../../../components/show';
 
-export const BEGRUNNELSE_MAX_LENGTH = 4000;
+export const BEGRUNNELSE_BEGRENSET_LENGTH = 4000;
+export const BEGRUNNELSE_MAX_LENGTH = 10000;
```

Comment:
Det er viktig at det er BEGRUNNELSE_MAX_LENGTH som blir brukt i skjema-utils.ts for validering og ikke BEGRUNNELSE_BEGRENSET_LENGTH.

Reply:
Det er allerede gjort. Jeg har ikke endret noe der.